Simulator/sun_sensor_math.py

```diff
@@ -74,7 +79,13 @@ def sense2vector(measurements, r_Earth2Sun, r_sat):
         sat2sun: satellite to sun 3-vector
     NOTE: This function can use numpy
     """
-    
+    if type(measurements) != list:
```

Comment:
Why do all of these have to be a list? A tuple would work just as well. Any iterable would be fine. I think the error here is "built in" since you can't index a non-iterable anyway.

Reply:
they don't necessarily, but are we using tuples in this case? The idea was to generate more helpful errors to isolate problems more quickly, though admittedly this one doesn't matter all that much.

Reply:
I personally think it adds clutter and that the error message will be no less cryptic when seen in context. It's up to you though.